### PR TITLE
feat(): Add mysql-terminal site

### PR DESCRIPTION
**Pull request by gilgda**
Add mysql-terminal site

### CODE REVIEW

File: src/content/guides/MYSQL/TERMINAL/TERMINAL.MD [hunk 1]
```diff
@@ -0,0 +1,23 @@
+MySQL - Terminal
+
+Schritt 1. Öffne das Terminal mit der Tastenkombination STRG + ALT + T.
+
+Schritt 2. Gib nun folgenden Konsolen-Befehl ein -> mysql -h 10.16.1.6 -u USERNAME -p
+
+![Screenshot 1](content/guides/MYSQL/TERMINAL/screen1.png)
+
+
```
Comment (Lordnoname) [inline src/content/guides/MYSQL/TERMINAL/TERMINAL.MD]: Please remove this empty lines

File: src/content/guides/MYSQL/TERMINAL/TERMINAL.MD [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Schritt 4. Gib nun dein Passwort von der Schule ein und bestätige wieder.
+
+![Screenshot 3](content/guides/MYSQL/TERMINAL/screen3.png)
+
+
+Schritt 5. Nun bist du am MySQL-Server eingeloggt und kannst MySQL Querys ausführen!
+
+
```
Comment (Lordnoname) [inline src/content/guides/MYSQL/TERMINAL/TERMINAL.MD]: Same as above

**Comment by DevVersion** [comment]
Beside the comments, LGTM

File: src/content/guides/MYSQL/TERMINAL/TERMINAL.MD (hunk 1 above)
Comment (gilgda) [inline src/content/guides/MYSQL/TERMINAL/TERMINAL.MD]: Already in work

File: src/content/guides/MYSQL/TERMINAL/TERMINAL.MD (hunk 2 above)
Comment (gilgda) [inline src/content/guides/MYSQL/TERMINAL/TERMINAL.MD]: Yes Sir

Comment (DevVersion) [inline src/content/guides/MYSQL/TERMINAL/TERMINAL.MD]: Sorry Mister, I'm the Sir

**Comment by Lordnoname** [comment]
@gilgda The assignment tells you who is currently checking your pr.

**Comment by DevVersion** [comment]
@Lordnoname @gilgda - Take over this PR :+1:

**Comment by DevVersion** [comment]
LGTM

**Pull request closed without merging**